src/views/hazardCharts/constants/hazardPageOptions.ts

```diff
@@ -206,11 +206,17 @@ export const hazardPageLocations: HazardPageLocations[] = [
     latitude: -39.59,
     longitude: 174.28,
   },
-  {
-    id: "srg_164",
+  // {
+  //   id: "srg_164",
```

Comment:
I thought we decided to continue using `srg_164` for the Franz Josef location?
```
>>> from nzshm_common.location import location_by_id
>>> location_by_id('WHO')
{'id': 'WHO', 'name': 'Franz Josef', 'latitude': -43.35, 'longitude': 170.17}
>>> location_by_id('srg_164')
{'id': 'srg_164', 'name': 'Franz Josef', 'latitude': -43.375755162, 'longitude': 170.18836548}
>>> 
```

Reply:
@chrisdicaprio @voj this is a temporary workaround. I guess I can revert it now. .....